module/VuFind/src/VuFind/View/Helper/Root/Citation.php

```diff
@@ -223,56 +262,71 @@ public function getDataCSL()
         default:
             $item['type'] = 'book';
         }
-        // subtitle -> shortTile
-        if (!empty($this->details['subtitle'])) {
-            $item['shortTitle'] = $this->details['subtitle'];
```

Comment:
Mapping subtitle to short title seems like the wrong thing to do, so I understand why you removed it... but do we need to use subtitle in a different field, or is it not needed at all?

Reply:
I didn't see support for subtitle in the schema. The schema has a few strange choices in it like that.